src/surforama/app.py

```diff
@@ -163,11 +167,9 @@ def _get_valid_image_layers(self, combo_box) -> List[Image]:
         ]
 
     def get_point_colors(self, points):
```

Comment:
Also added this linear interpolation here. The direct sampling on coordinates gave some artifacts, particularly on the Chlamy thylakoid demo example.